classifiers/reference_quality/chunked_sentence_complexity/README.md

```diff
@@ -0,0 +1,4 @@
+This is similar to the standard [sentence complexity brick](https://bricks.kern.ai/classifiers/972), with the difference that here, the text is chunked (split) before calculating the sentence complexity. With the standard sentence complexity brick, the accuracy of the complexity gets worse the longer a text is. Keep in mind that the difference is especially noticeable in really long texts, like whole news articles. The texts are split into sentences using SpaCy. After calculating the sentence complexities for each sentence, all complexities are aggregated again with a weighted average complexity.
```

Comment:
as far as i know, links aren't stable since the bricks id changes. if thats still the case we need a different approach for linking here

Reply:
True. For now, I see two options then: remove the link in general or link to the GitHub issue page: https://github.com/code-kern-ai/bricks/issues/2. Not ideal, though.

Reply:
I have removed the link for now.